### PR TITLE
3.1.5 release docs

### PR DESCRIPTION
**Pull request by HotelCalifornia**
```diff
+ release notes
+ function documentation associated with release
+ vision_zero* docs which were missing
- fix some misc spacing issues
- fix a couple of small typos
```

### CODE REVIEW

File: v5/api/cpp/vision.rst [hunk 1]
```diff
@@ -960,6 +1111,25 @@ that can be detected by the Vision Sensor
...
+  typedef enum vision_zero {
+    E_VISION_ZERO_TOPLEFT = 0,
+    E_VISION_ZERO_CENTER = 1
+  } vision_zero_e_t;
+
+======================= =============================================
+ Value
+======================= =============================================
+ E_VISION_ZERO_TOPLEFT   (0,0) coordinate is the top left of the FOV
+ E_VISION_ZERO_CENTER    (0,0) coordinate is the center of the FOV
```
Comment (Octogonapus) [inline v5/api/cpp/vision.rst]: This shouldn't be `(0, 0)`

Comment (HotelCalifornia) [reply v5/api/cpp/vision.rst]: are you saying it should be `(0, 0)`? I made them all `(0,0)` (no space) to be consistent with what's in the header files

Comment (Octogonapus) [reply v5/api/cpp/vision.rst]: Nope, I misunderstood the purpose of the enum.

**Comment by HotelCalifornia** [comment]
Ah, good call on those whitespace things. I changed the view in the online editor so that it would use two spaces instead of tabs, which fixed some things (but it looks like there were stray hard tabs that may have exploded or something)

**Pull request merged**